caffe2/CMakeLists.txt

```diff
@@ -260,13 +260,19 @@ endif()
 
 # ---[ Caffe2 HIP sources.
 if(USE_ROCM)
-  # Call again since Caffe2_HIP_INCLUDES is extended with ATen include dirs.
   IF(BUILD_ATEN)
+     # Call again since Caffe2_HIP_INCLUDES is extended with ATen include dirs.
      HIP_INCLUDE_DIRECTORIES(${Caffe2_HIP_INCLUDES})
+
+     # Necessary since some of ATen's C++ files contain HIP includes.
+     SET(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -D__HIP_PLATFORM_HCC__=1")
```

Comment:
wait is it hipify adding those includes?

Reply:
Nope. There's just Cpp files being generated by the build system that contain CUDA headers.